### PR TITLE
feat: add `autoSave` toggle button

### LINKED ISSUE #282: Add a switch button
Currently, `autoSave` is reactive, can we add a toggle button like `Show Error`?

image

### PR DESCRIPTION
**Pull request by btea**
close #282

### CODE REVIEW

File: src/Repl.vue [hunk 1]
```diff
@@ -63,14 +66,18 @@ if (!props.editor) {
   throw new Error('The "editor" prop is now required.')
 }
 
-const outputRef = useTemplateRef('output')
+const outputRef = useTemplateRef<typeof Output>('output')
```
Comment (sxzz) [inline src/Repl.vue:69]: Use the latest Volar, the generic type will be inferred automatically.

**Review by sxzz — CHANGES_REQUESTED** [review]

File: src/Repl.vue (hunk 1 above)
Comment (btea) [reply src/Repl.vue]: Thanks for the reminder, it has been updated.

**Review by sxzz — APPROVED** [review]
Minor tweaks

**Comment by btea** [comment]
Thanks!

**Pull request merged**